backend/Testing/Fixtures/IntegrationFixture.cs

```diff
@@ -37,6 +43,8 @@ private async Task DownloadTemplateRepo()
         await using var stream = await AdminApiTester.HttpClient.GetStreamAsync("https://drive.google.com/uc?export=download&id=1w357T1Ti7bDwEof4HPBUZ5gB7WSKA5O2");
         using var zip = new ZipArchive(stream);
         zip.ExtractToDirectory(_templateRepo.FullName);
+        InitRepoInDirectory(_templateRepo.FullName);
+        ZipFile.CreateFromDirectory(_templateRepo.FullName, _templateRepoZip.FullName);
```

Comment:
(Tim wrote that code, but he's asleep right now so I'll reply). The .zip file at that URL doesn't contain a `.hg` folder, so the InitRepoInDirectory step is necessary to turn it into an actual Mercurial repo.

However, I think it might be better to just add the `.hg` folder and then upload the result to Google Drive, or even just check it into out Git repo since the zip file is only 1.4 megabytes. (Hey, it would fit on a floppy disk!)

Reply:
Done yesterday; resolving conversation.